### ISSUE TITLE
Invisible model and continuous "Reloading local player model" when playing with a controller

### ISSUE THREAD

**Issue by webface-ca** [cmooref17/Lethal-Company-TooManyEmotes#140]
When I use an emote my player model goes invisible to myself but not other players.  Also found in the console log that upon entering the game, even before invoking an emote, that the message "Reloading local player model" starts to get logged right away and continuously.

If I unplug my controller and use a keyboard everything works as expected and that log message does not constantly get logged.

**Comment by cmooref17** [comment]
That's strange. Do you use inpututils? I believe I have a keybind for reloading the character model if anyone needs to do so. If you do have this mod, is it bound to anything on the controller?

**Comment by webface-ca** [comment]
No, I do not have InputUtils,  Should I?

I can do some playing around.  From what I recall I used the default controller config but rebound my LB to T for push to talk.  I did see some references to resetting the controller bindings to default and trying again so I may try that.

I'll play around a bit more with the controller bindings/settings to see if I can pinpoint anything.

**Comment by webface-ca** [comment]
I ended up disabling Steam controller input, resetting all key bindings (they were defaulted anyway) and no luck.

I then removed Too Many Emotes (as well as the Assets folder and config), restarted and reset the bindings again, then reinstalled TME, tried again and same issue.

I also tried removing Reserved Item Slot Core and Reserved Walkie Slot and retrying, but with same results.

NOTE: I did notice a NULL reference error thrown by the TME mod when I reset my key bind settings, if it helps.

FYI, the mods I am currently running are:

More Company
Faster Item Dropship
Helmet Cameras
Late Company
More Suits
Ship Loot
Reserved Item Slot Core
Reserved Walkie Slot
Too Many Emotes

All mods were installed fresh this past week as I was playing vanilla up until this point.
